index.html

```diff
@@ -0,0 +1,300 @@
+                    <img src="./assets/img/c1.png" alt="" class="clients__avatar">
+                    <div class="clients__user-info">
+                        <div class="clients__name">Regina Miles</div>
+                        <div class="clients__job">Designer</div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="team features container">
+        <div class="c-title">Meet Our Team</div>
+        <div class="c-subtitle c-subtitle--sm">Problems trying to resolve the conflict between 
+            the two major realms of Classical physics: Newtonian mechanics </div>
+        <div class="team__members l-items">
+            <div class="team__card">
+                <div class="team__img-container">
+                    <img src="./assets/img/t1.png" alt="" class="team__img">
+                </div>
+                <div class="team__infos">
+                    <div class="team__name">Ashley Fletcher</div>
+                    <div class="team__job">CEO</div>
+                    <div class="team__soc">
+                        <i class="fa-brands fa-square-facebook team__soc-icon"></i>
+                        <i class="fa-brands fa-instagram team__soc-icon"></i>
+                        <i class="fa-brands fa-twitter team__soc-icon"></i>
+                    </div>
+                </div>
+            </div>
+
+            <div class="team__card">
+                <div class="team__img-container">
+                    <img src="./assets/img/t2.png" alt="" class="team__img">
+                </div>
+                <div class="team__infos">
+                    <div class="team__name">Rodney Stratton</div>
+                    <div class="team__job">Consultant</div>
+                    <div class="team__soc">
+                        <i class="fa-brands fa-square-facebook team__soc-icon"></i>
+                        <i class="fa-brands fa-instagram team__soc-icon"></i>
+                        <i class="fa-brands fa-twitter team__soc-icon"></i>
+                    </div>
+                </div>
+            </div>
+
+            <div class="team__card">
+                <div class="team__img-container">
+                    <img src="./assets/img/t3.png" alt="" class="team__img">
+                </div>
+                <div class="team__infos">
+                    <div class="team__name">Avie Beaton</div>
+                    <div class="team__job">CO Founder</div>
+                    <div class="team__soc">
+                        <i class="fa-brands fa-square-facebook team__soc-icon"></i>
+                        <i class="fa-brands fa-instagram team__soc-icon"></i>
+                        <i class="fa-brands fa-twitter team__soc-icon"></i>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="contact container">
+        <div class="c-title c-title--xs c-title--left">Contact Us</div>
```

Comment:
Not a great idea to use CSS properties to name your bem classes with `c-title--xs` Something like `c-title--extra-small` sounds more descriptive.

Reply:
For someone who doesn't know TailwindCSS, it is confusing to read `c-title-xs` If you could be a little more descriptive with the class name here.